docs/04-operation-guides/troubleshooting/eventing/evnt-03-type-collision.md

```diff
@@ -27,4 +27,4 @@ Follow these steps to detect if naming collision is the source of the problem:
     kubectl get subscriptions.eventing.kyma-project.io -A | grep {CLEAN_EVENT_TYPE}
```

Comment:
Are you sure, this command is still relevant ? 

Also, you could rename the `{CLEAN_EVENT_TYPE}` to `{CLEAN_TYPE}`, since that's the result of the `kubectl -n {NAMESPACE} get subscriptions.eventing.kyma-project.io {NAME} -o jsonpath='{.status.types}'`: 
```
[{"cleanType":"order.paymentsuccess.v1","originalType":"order.payment*success.v1"}]%
```

Reply:
Updated.